File: csharp/entity/FlashEntity.cs
// Flash entity client for the Thesmsworks SDK.

using Voxgig.Struct;

namespace ThesmsworksSdk.Entity;

public class FlashEntity : ThesmsworksEntityBase
{
    public FlashEntity(ThesmsworksSDK client, Dictionary<string, object?>? entopts = null)
        : base(client, entopts, "flash")
    {
    }

    public override IEntity Make()
    {
        return new FlashEntity(client, CloneOpts());
    }

    // (load not defined by this API - base class throws UnsupportedOp)

    // (list not defined by this API - base class throws UnsupportedOp)

    // (create not defined by this API - base class throws UnsupportedOp)

    // (update not defined by this API - base class throws UnsupportedOp)

    // (remove not defined by this API - base class throws UnsupportedOp)
}

File: swift/Sources/ThesmsworksSdk/entity/FlashEntity.swift
// Flash entity client for the Thesmsworks SDK.

import Foundation

public final class FlashEntity: ThesmsworksEntityBase {
  public init(_ client: ThesmsworksSDK, _ entopts: VMap? = nil) {
    super.init(client, entopts, "flash")
  }

  public override func make() -> Entity {
    return FlashEntity(client, cloneOpts())
  }

  // (load not defined by this API - base class throws unsupportedOp)

  // (list not defined by this API - base class throws unsupportedOp)

  // (create not defined by this API - base class throws unsupportedOp)

  // (update not defined by this API - base class throws unsupportedOp)

  // (remove not defined by this API - base class throws unsupportedOp)
}
